packages/storage-plugin/schematics/src/ng-generate/keys-migration.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+import { workspaceRoot } from '@nrwl/devkit';
+import { join } from 'path';
+import { createWorkspace } from '../_testing';
+
+describe('Storage Plugin Migration', () => {
+  const ngxsSchematicRunner = new SchematicTestRunner(
+    '@ngxs/storage-plugin/schematics',
+    join(workspaceRoot, 'packages/storage-plugin/schematics/collection.json')
+  );
+
+  const testSetup = async () => {
+    const appTree = await createWorkspace();
+    appTree.overwrite('package.json', `{"dependencies": {"@ngxs/store": "3.8.2"}}`);
+
+    return { appTree };
+  };
+
+  it('migrate empty forRoot', async () => {
+    const { appTree } = await testSetup();
+
+    const newContent = `
+    import { NgxsStoragePluginModule } from '@ngxs/storage-plugin';
+    import { NgModule } from '@angular/core';
+    import { BrowserModule } from '@angular/platform-browser';
+
+    import { AppRoutingModule } from './app-routing.module';
+    import { AppComponent } from './app.component';
+
+      @NgModule({
+        declarations: [
+          AppComponent
+        ],
+        imports: [
+          BrowserModule,
+          AppRoutingModule,
+          NgxsStoragePluginModule.forRoot()
+        ],
+        bootstrap: [AppComponent]
+      })
+      export class AppModule { }
```

Comment:
can you please add cases like this 
```typescript
const imports = [
  NgxsStoragePluginModule.forRoot()
];


@Module({

imports,

})

export class MyModule { ... }
```

along with support for standalone applications

Reply:
I added a unit test for this case

```ts
const imports = [
  NgxsStoragePluginModule.forRoot()
];

@NgModule({

imports,

})

export class MyModule { ... }
```
The `imports` declaration is covered by the schematic since this ` NgxsStoragePluginModule.forRoot()` falls under the `CallExpression` and `PropertyAccessExpression`

Reply:
As for the standalone applications, I was under the impression that the standalone will be released with 3.9.
@arturovt can you please advise?